Multisig/UI/UI Library/Views/AddressInfoView.swift

```diff
@@ -190,9 +208,14 @@ class AddressInfoView: UINibView {
 
 extension UIViewController {
     @objc func didTapAddressInfoDetails(_ sender: AddressInfoView) {
-        if let url = sender.browseURL {
-            openInSafari(url)
+        if sender.showDetailNavigation,
+           let keyInfo = try? KeyInfo.firstKey(address: sender.address) {
+                let vc = OwnerKeyDetailsViewController(keyInfo: keyInfo)
+                show(vc, sender: self)
+        } else {
+            if let url = sender.browseURL {
+                openInSafari(url)
+            }
```

Comment:
this is not how this should be implemented. Instead, we need to use a simple table view row with accessory input, and put the address there. We should have examples of such thing in several places in the interface.